src/components/objectIdentifierCard/objectIdentifier.component.tsx

```diff
@@ -11,15 +13,19 @@ import { getObjectDisplayName } from "src/lib/utils";
 interface ObjectIdentifierCardProps {
   object: ParsedSkylarkObject;
   children?: ReactNode;
-  onForwardClick?: (o: SkylarkObjectIdentifier) => void;
   disableForwardClick?: boolean;
+  disableDeleteClick?: boolean;
+  onForwardClick?: (o: SkylarkObjectIdentifier) => void;
+  onDeleteClick?: () => void;
 }
 
 export const ObjectIdentifierCard = ({
   object,
   children,
   disableForwardClick,
+  disableDeleteClick,
```

Comment:
thinking if we really need the delete word here.
`disableDelete`
`onDelete`
... etc seem enough

Reply:
true